libdevcore/SHA3.cpp

```diff
@@ -192,6 +199,14 @@ static inline int hash(uint8_t* out, size_t outlen,
 	if (outlen > (bits/8)) {                                      \
 	  return -1;                                                  \
 	}                                                             \
+	return hash(out, outlen, in, inlen, 200 - (bits / 4), 0x06);  \
```

Comment:
It is hard to find any of these (couldn't easily spot it in the spec), but on the bottom of the Keccak site there's a summary: http://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.202.pdf